clients/cpp-console/include/MicMuter.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/// <summary>
+/// Abstract object used to define the interface to a MicMuter
+/// </summary>
+/// <remarks>
+/// </remarks>
+class IMicMuter
+{
+public:
+    /// <summary>
+    /// The destructor should be defined to clean up any variables or resources.
+    /// </summary>
+    /// <remarks>
+    /// </remarks>
+    virtual ~IMicMuter() = default;
+
+    /// <summary>
+    /// Initialize will initialize the MicMuter with any specific OS dependent 
+    /// settings. If called without parameters it should assume some appropriate
+    /// defaults.
+    /// </summary>
+    /// <returns>A return code with S_OK as success and any other int as an error</returns>

```

Comment:
S_OK is Windows specific. Say that a return value of 0 is success, and another other value is faulure

Reply:
Updated.